website/docs/docs/user-interface/bi-directional.md

```diff
@@ -94,4 +94,12 @@ When provider and consumer contracts are incompatible, incompatibility messages
 | Response body is incompatible | Pact response's body content is incompatible with provider contract |
 | Response body contains unknown information | Pact response's body contains unknown content compared with provider contract|
 | Response request is incompatible | Pact response's request content is incompatible with provider contract|
-| Response header contains unknown information | Pact response's header contains unknown content compared with provider contract |
\ No newline at end of file
+| Response header contains unknown information | Pact response's header contains unknown content compared with provider contract |
+| Response body is incompatible with the response body schema in the spec file: must NOT have unevaluated properties | Pact response body contains properties that don't match one of the schemas defined in an `allOf` declaration |
+| Request body is incompatible with the request body schema in the spec file: must NOT have unevaluated properties | Pact request body contains properties that don't match one of the schemas defined in an `allOf` declaration |
```

Comment:
Come to think of it, I only did this for *responses*. We'd need to for `unevaluatedProperties`, right? We don't need to do anything with `additionalProperties` or `required` mangling. I believe

Reply:
Yes, I think we do need to do that.

Reply:
Closing as https://github.com/pactflow/swagger-mock-validator/pull/34/files will resolve this.